src/helpers/misc/Animal.re

```diff
@@ -323,7 +323,11 @@ let getOrgBadgeImage: t => string =
     // | Tarkus
     // | _ => None
     };
-
+[
```

Comment:
@moose-code You didn't catch this :D 

I was using this as a copy pad for the new graphql backend (that will store all this data).

This shouldn't have been in this PR; I apologies.